vscode-extension/src/extension.ts

```diff
@@ -756,53 +758,42 @@ async function setupEnvironmentVariables() {
   }
 
   if (fs.existsSync(envPath)) {
-    // Append some lines in code to help guide users
-    // https://ai.google.dev/tutorials/setup
-    // https://platform.openai.com/api-keys
-    // Also add HF one
     vscode.window.showInformationMessage(
       "Env file already exists, will implement next PR"
     );
   } else {
-    // File doesn't exist in path
-    vscode.window.showInformationMessage(
-      "Env file does not already, will implement next PR"
+    // Create the .env file from the sample
+    const envTemplatePath = vscode.Uri.joinPath(
+      context.extensionUri,
+      "static",
+      "env_template.env"
     );
-  }
 
-  // // Create the model registry file from the sample
-  // const sampleModelRegistryPath = vscode.Uri.joinPath(
-  //   context.extensionUri,
-  //   "static",
-  //   "example_aiconfig_model_registry.py"
-  // );
+    try {
+      await vscode.workspace.fs.copy(
+        envTemplatePath,
+        vscode.Uri.file(envPath),
+        { overwrite: false }
+      );
+    } catch (err) {
+      vscode.window.showErrorMessage(
+        `Error creating new file ${envTemplatePath}: ${err}`
+      );
+    }
 
-  // try {
-  //   await vscode.workspace.fs.copy(
-  //     sampleModelRegistryPath,
-  //     vscode.Uri.file(modelRegistryPath),
-  //     { overwrite: false }
-  //   );
-  // } catch (err) {
-  //   vscode.window.showErrorMessage(
-  //     `Error creating new file ${modelRegistryPath}. Error is ${err}`
-  //   );
-  // }
-
-  // const doc = await vscode.workspace.openTextDocument(modelRegistryPath);
-  // if (doc) {
-  //   vscode.window.showTextDocument(doc);
-  //   vscode.window.showInformationMessage(
-  //     "Please customize your new model registry."
-  //   );
-  // }
-
-  // let config = vscode.workspace.getConfiguration(EXTENSION_NAME);
-  // await handleCustomModelRegistryUpdate(
-  //   config,
-  //   aiconfigEditorManager,
-  //   modelRegistryPath
-  // );
+    const doc = await vscode.workspace.openTextDocument(envPath);
+    if (doc) {
+      vscode.window.showTextDocument(doc, {
+        preview: false,
+        // Tried using vscode.ViewColumn.Active but that overrides existing
+        // walkthrough window
+        viewColumn: vscode.ViewColumn.Beside,
+      });
+      vscode.window.showInformationMessage(
+        "Please define your environment variables."
+      );
+    }
+  }
```

Comment:
nit: what if it doesn't open?

Reply:
Good catch, will follow up someday

Reply:
https://github.com/lastmile-ai/aiconfig/issues/1287